build/buildRuby.sh

```diff
@@ -202,6 +202,9 @@ export PATH=${RUBY_DESTDIR}/bin:$PATH
 echo "Installing Bundler into Ruby ..."
 elevate ${RUBY_DESTDIR}/bin/gem install ${BASE_DIR}/source/ext/gems/bundler-1.10.6.gem
 
+echo "Installig Mysql2 into Ruby ..."
```

Comment:
Also, you changed the build process for Ruby, but you did not bump the directory in /usr/local. This will break unit tests. If you change this file, it's highly likely that you need to bump the directory. In your case, it's mandatory.